### ISSUE TITLE
Not compatible with Nextcloud 16

### ISSUE THREAD

**Issue by schniepp** [nextcloud/deck#996]
**Describe the bug**
Deck is completely broken in NC16, when it runs perfectly fine in NC15.

**To Reproduce**
1. Install NC16 from scratch
2. Add "Deck" app
3. Open "Deck" app to see that UI does not look as expected and is useless.

**Expected behavior**
Deck UI should just come up as expected and look at least similar to what we have in NC15

**Screenshots**

**Client details:**
 - OS: ubuntu 16.04
 - Browser: chrome
 - Version: Deck 0.5.2, NC 16-beta2
 - Device: desktop

<details>
<summary>Server details</summary>

**Operating system**:

**Web server:**

**Database:**

**PHP version:**

**Nextcloud version:** (see Nextcloud admin page)

**Where did you install Nextcloud from:**

**Signing status:**

```
Login as admin user into your Nextcloud and access
http://example.com/index.php/settings/integrity/failed
paste the results here.
```

**List of activated apps:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```

**Nextcloud configuration:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or

Insert your config.php content here
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

</details>

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```

</details>

**Comment by jospoortvliet** [comment]
Yeah, same here, the UI looks fairly OK but one can't do much in it... You can't create or rename boards. The only button that works is the settings, which pop up - and then do weird, too :dagger: 
![Screenshot_20190414_180007](https://user-images.githubusercontent.com/551757/56095647-33208d00-5edf-11e9-8a10-af7aeae387a8.png)

**Comment by schniepp** [comment]
I tried installing the new 0.6.0-beta1 today, hoping it would introduce nc16 compatibility. However, something strange happened. It first looked like it would let me install 0.6.0-beta1, but after a refresh of the Apps page, the 0.5.2 version showed up again. Observed behavior of the app was the same as reported above, not working. Strangely, this even happened when I first uninstalled 0.5.2 and made sure the "deck" folder was gone from the "apps" folder in my nc installation.
